jkube-kit/jkube-kit-spring-boot/src/main/java/org/eclipse/jkube/springboot/generator/LegacyJarGenerator.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.springboot.generator;
+
+import org.eclipse.jkube.generator.api.GeneratorConfig;
+import org.eclipse.jkube.generator.api.GeneratorContext;
+
+public class LegacyJarGenerator extends AbstractSpringBootNestedGenerator {
```

Comment:
What does `LegacyJar` mean in this context? (In Quarkus, legacy-jar is a Quarkus-specific packaging mode)

Can we give it a more relevant name.
If the purpose of using the nested-generator approach is to provide a follow-up SpringBootNativeGenerator, then we might want to call this one SpringBootJvmGenerator.
If instead the purpose is to provide different generators for different Spring Boot versions, then we might want to find something more suitable and descriptive too.

Reply:
It means jar which doesn't contain layered format (missing `BOOT-INF/layers.idx`)

I can rename it to StandardJar but since Springboot 2.3.x layered jar seems to be the standard

Reply:
FatJar seems like the better description then

Reply:
(fat, uber...)